__tests__/materials/material.mutation.test.js

```diff
@@ -0,0 +1,440 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-test-client');
+const {
+  MATERIAL_ALREADY_EXIST,
+  MATERIAL_NOT_FOUND,
+} = require('../../error-messages/material.messages');
+const {
+  user,
+  material,
+  materialToUpdate,
+  materialWithExistingName,
+} = require('./material.variables');
+const { adminLogin } = require('../helper-functions');
+
+let token;
+let materialId;
+const materialDoesNotExistId = '1f2ad410eb01783384e6111b';
+
+describe('test material mutations', () => {
+  beforeAll(async () => {
+    token = await adminLogin(user);
+  });
+  test('#1 should add material to database', async () => {
```

Comment:
Please, create 3 `describe` statements for `Create` tests, `Update` tests, `Delete` tests to increase the readability of the tests

Reply:
fixed